scanpipe/pipelines/deploy_to_develop.py

```diff
@@ -81,6 +82,11 @@ def steps(cls):
         ".soy",
         ".class",
     ]
+    uninteresting_extensions = [
+        ".png",
+        ".gif",
+        ".svg",
+    ]
```

Comment:
Why those 3 images format in particular?

Reply:
The list of uninteresting extensions is not yet final. We will need input from @pombredanne  and @AyanSinhaMahapatra  on which file types do not require a `requires-review` status.

Reply:
I would rather not list extensions but use the `is_media`  or similar flags from SCTK in the future..... for now, if you are trying to tag images, do not forget JPEG.

Reply:
Ack, we are now using `is_media` flag insted.
Leftover non media files will marked as `requires-review`